src/Products/Comparison/Service/ComparisonServiceImpl.cs

```diff
@@ -243,7 +237,7 @@ private ICompareResult CompareFiles(CompareRequest compareRequest)
 
             if (compareResult == null)
             {
-                throw new Exception("Something went wrong. We've got null result.");
+                throw new NullReferenceException("Something went wrong. We've got null result.");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['System.NullReferenceException' should not be thrown by user code.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Comparison-for-.NET-MVC/pullRequest?prid=5038425)